include/mgp.hpp

```diff
@@ -3430,14 +3444,12 @@ inline mgp_type *Return::GetMGPType() const {
   return util::ToMGPType(type_);
 }
 
-void AddProcedure(mgp_proc_cb callback, std::string_view name, ProcedureType proc_type,
-                  std::vector<Parameter> parameters, std::vector<Return> returns, mgp_module *module,
-                  mgp_memory *memory) {
-  auto proc = (proc_type == ProcedureType::Read) ? mgp::module_add_read_procedure(module, name.data(), callback)
-                                                 : mgp::module_add_write_procedure(module, name.data(), callback);
-
+// do not enter
+namespace detail {
+inline void AddParamsReturnsToProc(mgp_proc *proc, std::vector<Parameter> &parameters,
```

Comment:
It needs to be inlined as the whole file is just inlined. This is how it is written since we have it in the header file. If it is not inlined there were bugs

Reply:
Ah, actually sorry I was dumb. Clang was complaining about ODR violations, so I used inline, I will remove it.